src/style.css

```
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.search {
+  display: flex;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  margin: 20px 0;
+}
+
+.search--big {
+  height: 70px;
+}
+
+.search--small {
+  height: 42px;
+}
```

```suggestion
.search {
  display: flex;
  box-sizing: border-box;
  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  margin: 20px 0;
    height: 70px;
}

.search--small {
  height: 42px;
}
```
FYI: you could also do it in this way. In common `search` (and other classes), w/o any modificators (like `--big`) set all values needed. In classes with modificator (for small) overwrite existing values that you need. It'll allow you to get rid of part of the classes in HTML

But IMO, the way you did is acceptable, so change it or not up to you